### ISSUE TITLE
I can't block url or ads

### ISSUE THREAD

**Issue by audiopsicotiko** [2dust/v2rayNG#1200]
hi, in the latest versions of v2rayNG "custom rules" does not work. Specifically, the "url blocking" does not work but I can block an IP. Geosite: category-ads-all doesn't work either.

test on android 7.1 android 10 and 11.

**Comment by ghost** [comment]
首先，你需要勾选客户端内的[启用流量探测]+[启用本地DNS]，再将[域名策略]设置为IPIfNonMatch(推荐)或IPOnDemand。

如果你要阻止某网站，可在[自定义规则]>[阻止的网址或IP]添加“domain:阻止的域名”。例如需要阻止google，就是“domain:google.com”。

直接引用geosite/geoip的前提是你客户端内的geo资源文件里有与之相对应的配置。有关geo资源文件的具体说明：https://github.com/2dust/v2rayNG

**Comment by audiopsicotiko** [comment]
> 首先，你需要开启客户端内的[启用流量探测]+[启用本地DNS]。
> 
> 如果你要阻止某网站，可在[自定义规则]>[阻止的网址或IP]添加“domain:阻止的域名”。例如需要阻止google，就是“domain:google.com”。
> 
> 直接引用geosite/geoip的前提是你客户端内的geo资源文件里有与之相对应的配置。有关geo资源文件的具体说明：https://github.com/2dust/v2rayNG


I appreciate your response.
I have managed to block the domains! follow your steps.

Regarding the blocking of advertising, I have downloaded the "geosite.dat" from https://github.com/2dust/v2rayNG, I have imported it to v2rayng and then I have managed to block the advertising with "geosite: category-ads-all"

everything has worked perfectly, thank you very much !!!
